### PR TITLE
Change hash algorithm from argon2 to doubl sha256

### PR DESCRIPTION
**Pull request by outsideris**
### Github Issue
Replace #431 

### Background
Argon2 use too many CPU resources. As @zzim2x 's testing, it took about 5 secs for hashing in browser side. 

### Solution
Use double sha256 hash like bitcoin.

argon 2 benchmark:
```
2000000000           0.03 ns/op
PASS
```

double sha256 banchmakr:
```
2000000000           0.00 ns/op
PASS
```

### CODE REVIEW

**Comment by outsideris** [comment]
I gave up to write integration test after I tried several times.
Please save me. @Geod24 @spikeekips :)

**Comment by Charleslee522** [comment]
Please check #399

**Comment by outsideris** [comment]
@Charleslee522 I didn't understand what I should check in #399 
This PR changed hash, so the signature should be updated.

**Comment by Charleslee522** [comment]
@outsideris Oh, does integration_tests fails because of signature?
I think it because of performance.

**Comment by spikeekips** [comment]
Argon2 makes also problem in client side :(

**Comment by spikeekips** [comment]
Anyway, I can make proper tx message with new hashing algo.

**Comment by spikeekips** [comment]
@outsideris I generate the overdraft/*.json with your code. Check https://gist.github.com/spikeekips/0002b08c6f83ea80c300eb4e5125018a .

**Comment by spikeekips** [comment]
@outsideris I used this tool to sign again. https://github.com/spikeekips/sebak-tools/tree/master/sebak-tx-sign

**Comment by outsideris** [comment]
Finally, I passed CI!!

File: docker/entrypoint.sh [hunk 1]
```diff
@@ -12,5 +12,5 @@ if [ $# -gt 0 ]; then
     exec ./sebak $@
 else
     # Node mode
-    exec ./sebak node --genesis=${SEBAK_GENESIS_BLOCK}
+    exec ./sebak node --genesis=${SEBAK_GENESIS_BLOCK},${SEBAK_COMMON_ACCOUNT}
```
Comment (Geod24) [inline docker/entrypoint.sh]: What does the common account has to do with this ?

File: tests/overdraft/request_1_2821_create_account_1.json [hunk 2]
```diff
@@ -3,12 +3,12 @@
...
     "version": "",
     "created": "2018-09-27T16:26:22.830373928+09:00",
-    "signature": "5TVx3JQHgZhHGcNLAyLfJQfssu4Hh7QRWBfjymEd7i1yC2s3yjSjKs92nwdkupKjFsQdbUNZHdkSQhicrT8ZSVWz"
+    "signature": "44unZEoQy41vYQr8LCFwJfXv8FpM6waxddwDd4sAs7Koo1XiJy18Lvp2FH6JG18DbvMwJTs5VQnoVoNEyJdZqicM"
   },
   "B": {
     "source": "GDIRF4UWPACXPPI4GW7CMTACTCNDIKJEHZK44RITZB4TD3YUM6CCVNGJ",
     "fee": "10000",
-    "sequenceID": 0,
+    "sequenceid": 0,
```
Comment (Geod24) [inline tests/overdraft/request_1_2821_create_account_1.json]: Why is this changed ?

Comment (outsideris) [reply tests/overdraft/request_1_2821_create_account_1.json]: I think it caused when I print the body and signature with [sebak-tx-sign](https://github.com/spikeekips/sebak-tools/tree/master/sebak-tx-sign). Indeed, there was a reason.
BTW, it does not affect the signature.

Comment (Geod24) [reply tests/overdraft/request_1_2821_create_account_1.json]: And sebak deserialize it okay, I suppose ? Anyway, since it has nothing to do with changing that algorithm, it should not have been changed here.

Comment (outsideris) [reply tests/overdraft/request_1_2821_create_account_1.json]: > And sebak deserialize it okay, I suppose ?

I think so. Yes, it was my mistake. I was too happier when integration tests are passed. If it causes any problems, it can be reverted.
